csunplugged/topics/content/en/sorting-networks/unit-plan/lessons/investigating-variations-using-the-sorting-network.md

```diff
@@ -87,12 +87,12 @@ Have the students try the Sorting Network with some of these words (note that yo
 There is also a number of words that have the letters in reverse alphabetical order, such as SPONGE and ZONKED (these can be sorted using the "larger to the left" variation, or can be read from the far side of the Sorting Network).
 Some words with this property have double letters in them, such as BELLOW; these will sort correctly, since the order of the double letters is immaterial.
 
-{image file-path="img/topics/sorting-network-toffees-cellos-sponge.png"}
-
 {panel type="general"}
 
 # List of words with letters in alphabetical order
 
+{image file-path="img/topics/sorting-network-toffees-cellos-sponge.png"}
```

Comment:
What are you thoughts on having some text before the image? I think the text is more important than the image.